__tests__/integration-tests/routes/news.test.js

```diff
@@ -7,6 +7,8 @@ const regions = require('../../../src/data/regions.json');
 const app = createServer();
 const request = supertest(app);
 
+describe('News Api Route', () => {
+  
 describe('News Api Route', () => {
```

Comment:
This could be confusing on the test panel, so I think we should use a different description for the separate block. Something like `'News All Articles (Main) Route'` I think would be fitting.

Reply:
Did changes . Please have a look